src/homework5/task2.py

```diff
@@ -0,0 +1,40 @@
+# Создайте декоратор, который хранит результаты вызовов функции
+# (за все время вызовов, не только текущий запуск программы)
+import os.path
+
+
+def dec(func):
+
+    def wrapper(*args, **kwargs):
+        dct = {}
+        func_name = func.__name__
+        if os.path.exists('text.txt'):
+            with open('text.txt', 'r') as fh:
+                for line in fh:
+                    key, value = line.split()
+                    dct[key] = value
+
+        dct[func_name] = str(int(dct.get(func_name, "0")) + 1)
+        print(func_name + " количество вызовов: " + dct[func_name])
```

Comment:
doesn't solve the task in the description. should keep call results, not number of calls